### PR TITLE
Grab bag of fixes

### PR DESCRIPTION
**Pull request by dwaynebailey**
This is a grab bag of fixes from mlo.  I've tried to commit them as distinct things without trying to make PRs for everything.  Would appreciate some review though.

### CODE REVIEW

File: translate/convert/accesskey.py [hunk 1]
```diff
@@ -135,8 +135,8 @@ def extract(string, accesskey_marker=DEFAULT_ACCESSKEY_MARKER):
             if (accesskey_marker == '&' and
                 XMLEntityPlaceable.regex.match(string[marker_pos-1:])):
                 continue
-            label = string[:marker_pos-1] + string[marker_pos:]
-            if string[marker_pos] != " ":  # FIXME weak filtering
+            if string[marker_pos] != " ":  # FIXME week filtering
```
Comment (unho) [inline translate/convert/accesskey.py]: week filtering? What does this mean?

**Comment by unho** [comment]
Some of the changes in po2dtd seem to require further tests tweaks for Travis to pass.

File: translate/convert/accesskey.py (hunk 1 above)
Comment (dwaynebailey) [inline translate/convert/accesskey.py]: Mmm that doesn't make sense at all :/

**Comment by unho** [comment]
Perhaps it is a good idea to try to break this into smaller chunks so some stuff can be landed.

File: translate/storage/pypo.py [hunk 2]
```diff
@@ -353,7 +353,7 @@ def addnote(self, text, origin=None, position="append"):
             commentlist = self.automaticcomments
             linestart = "#. "
         text = text.split("\n")
-        newcomments = [linestart + line + "\n" for line in text]
+        newcomments = [linestart + line[:-1] + line[-1:].replace("\\", "\\ ") + "\n" for line in text]
```
Comment (unho) [inline translate/storage/pypo.py]: Some test would be good to have.

**Comment by dwaynebailey** [comment]
We've landed all but the `\` at end of line in PO files into `master`. So closing this

**Pull request closed without merging**